### PR TITLE
History entry added to cube after running sumspice. Fixes #4972.

### PR DESCRIPTION
**Pull request by kaitlyndlee**
## Description
When running sumspice on a cube, a history entry for sumspice was previously not added to the cube's history. Now, a history entry is added. 

## Related Issue
https://isis.astrogeology.usgs.gov/fixit/issues/4972

## Motivation and Context
A history entry should be added for all applications ran on a cube.

## How Has This Been Tested?
This change was tested by running sumspice on a cube provided by Tammy Becker, running cathist on it, and checking to see if a history entry was added.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/control/apps/sumspice/SumFinder.cpp [hunk 1]
```diff
@@ -44,6 +44,8 @@
 #include "Kernels.h"
 #include "NaifStatus.h"
 #include "Progress.h"
+#include "History.h"
+#include "Application.h"
```
Comment (SgStapleton) [inline isis/src/control/apps/sumspice/SumFinder.cpp:48]: Was this import being used and where?

**Comment by SgStapleton** [comment]
I had the one question, but other than that this looks like a straightforward change.

**Comment by kaitlyndlee** [comment]
I am sorry, but I am not seeing another comment from you with your question.

**Comment by SgStapleton** [comment]
Just the comment on SumFinder.cpp about the Application.h inclusion.

**Comment by kaitlyndlee** [comment]
SumFinder.cpp needs the Application.h inclusion for line 714.

**Review by SgStapleton — APPROVED** [review]
This looks good!

**Pull request merged**